docs/luigi-core-api.md

```diff
@@ -416,6 +416,128 @@ Type: [Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Globa
 -   `scope` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** scope, can be empty if it is not required
 -   `idToken` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** id token, used for renewing authentication
 
+# &lt;&lt;&lt;&lt;&lt;&lt;&lt; HEAD
+
+#### login
+
+Login the user dynamically.
+This will run the same functionality as though the user clicked the login button.
+
+##### Examples
+
+```javascript
+Luigi.auth().login();
+```
+
+**Meta**
+
+-   **since**: NEXTRELEASE
+
+> > > > > > > cc8090040fa22aef1a9eeeb45f0f12db9d485c08
```

Comment:
I guess you had some merge conflicts. Some docu parts are doubled and also these less- and greater-than signs.

Reply:
cleaned